ChallengeMk2/Controls/PopupLabel.cs

```diff
@@ -0,0 +1,152 @@
+
+using Xamarin.Forms;
+
+namespace ChallengeMk2.Controls
+{
+    public class PopupLabel : ContentView
```

Comment:
I do not used Xaml because i do not know how to do animation in xaml. 
But i just find that free microsoft eBook, chapter 22 on Animations. I will read that and i will make a try with xaml ;)
But i am not sure how to do animation within xaml without use of event. I have to find out.

Reply:
I meant use XAML for creating the view.  Then you can give x:name to your view elements and perform your animations in the code-behind.

Reply:
oki i see. I do that

Reply:
Add new custom control that uses xaml to define view.